internal/resources/const.go

```diff
@@ -67,7 +67,6 @@ const (
 
 var DefaultDragonflyArgs = []string{
 	"--alsologtostderr",
-	"--primary_port_http_enabled=false",
```

Comment:
why? better to scrape from admin so that we separate all these tasks?

Reply:
In that case, we need an admin service, right?

Reply:
Why? 

Looking at the service monitor, we are scraping the pods using the selectors right? changing the port to be `9999` should do it. Can you check if that works?

Reply:
serviceMonitors monitors services, so as far as I understand, it targets services and scrap data through it. I am checking though